docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -91,19 +91,40 @@ This analysis can be categorised as relating
 to the energetics of the different lambda states that were sampled,
 or to the analysis of the change in structural conformation over time in each state.
 
-* lambda energetic analysis
-
-  * free energy overlap matrix
-  * replica exchange overlap matrix (for repex only)
-  * timeseries of replica states
-  * forward and reverse estimates of free energies
-
-* relating to structural analysis. If a protein was present, these analyses first center and align the system so that
+.. todo: issue 792, consolidate this page into its own analysis page and link both RBFE and AFE pages to it
+
+* Energetic and replica exchange analysis.  These analyses consider the swapping and energetic overlap between the
+  different simulated states to try to establish the convergence and correctness of the estimate of free energy
+  difference produced.
+
+  * MBAR overlap matrix.  This plot is used to assess if the different lambda states simulated overlapped energetically.
+    Each matrix element represents the probability of a sample from a given row state being observable in a given column
+    state.
+    This plot should show that the diagonal of the matrix has some "width" so that the two end states are connected.
+  * Replica exchange overlap matrix (for repex type simulations only).  Similar to the MBAR overlap matrix, this shows
+    the probability a given lambda state exchanged with another.  Again, the diagonal of this matrix should be at least
+    tridiagonal wide for the two end states to be connected.
+  * timeseries of replica states.  This plot shows the evolution of time of the different lambda states as they are
+    exchanged between different configurations.
```

Comment:
I would write it the other way around: This plot shows the time evolution of how the simulation trajectories traverse through lambda space as configurations are exchanged between different lambda states. (or something like that)